src/snowflake/snowpark/_internal/analyzer/datatype_mapper.py

```diff
@@ -213,9 +213,33 @@ def schema_expression(data_type: DataType, is_nullable: bool) -> str:
         else:
             return "to_timestamp('2020-09-16 06:30:00')"
     if isinstance(data_type, ArrayType):
+        if data_type.structured:
+            element = schema_expression(data_type.element_type, is_nullable)
+            return f"to_array({element}) ::" + convert_sp_to_sf_type(data_type)
```

Comment:
```suggestion
            return f"to_array({element}) ::{convert_sp_to_sf_type(data_type)}"
```
why not use this? also same below